poorwsgi/request.py

```diff
@@ -797,21 +800,23 @@ def getfirst(self, key, default=None, fce=str):
         if val is None:
             return None
 
-        if isinstance(val, list):
+        if isinstance(val, (list, set, tuple)):
```

Comment:
you could use here `collections.Iterable` for checking:

https://stackoverflow.com/a/19919392